crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -1992,6 +2011,17 @@ impl OlmMachine {
         self.inner.identity_manager.update_tracked_users(users).await
     }
 
+    /// Mark all tracked users as dirty.
```

Comment:
`SyncedKeyQueryManager` is not `pub` it's `pub(crate)` so people can't look at the docs for this.

Shame that we can't put this method anywhere else since it needs the `IdentityManager`. Later on we should probably expose the `IdentityManager` publicly and move all the tracked user methods under that struct.